### PR TITLE
Fix linter warnings

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem
- *Description of why you made this PR*

## Solution
- *And how do you fix that problem*

## PR Status
- [ ] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Package_check results
---
*If you have access to [App Continuous Integration for packagers](https://yunohost.org/#/packaging_apps_ci) you can provide a link to the package_check results like below, replacing '-NUM-' in this link by the PR number and USERNAME by your username on the ci-apps-dev. Or you provide a screenshot or a pastebin of the results*

[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/REPLACEBYYOURAPP_ynh%20PR-NUM-%20(USERNAME)/)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by benneti** [comment]
thanks, can you rerun the tests?

**Comment by ericgaspar** [comment]
Yes sure, the test is queuing but at the moment the CI dev is broken: https://ci-apps-dev.yunohost.org/ci/

**Comment by benneti** [comment]
ah was already wondering about that, as you might have seen in the initial PR.
No problem then we can also wait until the ci works again.
Do you know how to get access to the testme command of the CI?

File: check_process [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+		sudo yunohost app install https://github.com/YunoHost-Apps/synapse_ynh/ -a "domain=$domain&server_name=$server_name&is_public=$is_public&jitsi_server=$jitsi_server" --force
+	; Manifest
+		port="8449"
+		synapsenumber="1"
+		botname="telegrambot"
+		encryption=false
+		botadmin="@johndoe:synapsedomain.tld"
+		botusers="@johndoe:synapsedomain.tld"
+		apiid="12345"
+    # this is a random hash so this will not work
```
Comment (benneti) [inline check_process:13]: can it be that this comment leads to problems?

File: check_process [hunk 2]
```diff
@@ -0,0 +1,36 @@
+;; Default test serie
+	; pre-install
+		sudo yunohost tools update apps
+		sudo yunohost app install https://github.com/YunoHost-Apps/synapse_ynh/ -a "domain=$domain&server_name=$server_name&is_public=$is_public&jitsi_server=$jitsi_server" --force
+	; Manifest
+		port="8449"
```
Comment (benneti) [inline check_process:6]: I think we do not need this

**Review by benneti — COMMENTED** [review]
I just looked at the logs again, and I the problem is that a real api_id is necessary for the configuration step to work or maybe at least one with the correct length like "1234567".
The hash at least has the correct length, do you mind changing this and reruninng the test?

**Comment by ericgaspar** [comment]
!testme

**Comment by benneti** [comment]
weird now it was telling me that python3-venv should be installed. Can we try again?

BTW whoom/where do I need to ask to be able to use testme?

**Comment by benneti** [comment]
OK this error I do not understand at all, is there a way to get the complete log not only what is found here https://ci-apps-dev.yunohost.org/ci/job/3084 ?

**Comment by ericgaspar** [comment]
complete logs are here: https://ci-apps-dev.yunohost.org/ci/logs/mautrix_telegram_amd64_stable_complete.log

**Comment by benneti** [comment]
I don't get it, the only thing I can imagine is that matrix is missing in the CI container, can I somehow install it or is it better to tweak the scripts to not rely on it?

**Pull request closed without merging**